add-permissions.js

```diff
@@ -13,6 +13,18 @@ module.exports = class AwsAddLambdaAccountPermissions {
     this.hooks = {
       'package:createDeploymentArtifacts': () => this.beforeDeploy(),
     };
+    this.serverless.configSchemaHandler.defineFunctionProperties(this.provider, {
```

Comment:
I think we need to check whether `configSchemaHandler` is defined as older serverless versions, probably, do not have it defined.

Reply:
fixed 😄